plugins/woocommerce-admin/client/marketplace/components/my-subscriptions/my-subscriptions.tsx

```diff
@@ -38,13 +39,24 @@ export default function MySubscriptions(): JSX.Element {
 		},
 		''
 	);
-	const updateConnectionHTML = sprintf(
-		// translators: %s is a link to the update connection page.
+
+	const installedTableDescription = createInterpolateElement(
 		__(
-			'If you don\'t see your subscription, try <a href="%s">updating</a> your connection.',
+			'WooCommerce.com extensions and themes installed on this store. To see all your subscriptions go to <a>your account<custom_icon /></a> on WooCommerce.com.',
 			'woocommerce'
 		),
-		updateConnectionUrl
+		{
+			a: (
+				<a
+					href="https://woocommerce.com/my-account/my-subscriptions"
+					target="_blank"
+					rel="nofollow noopener noreferrer"
+				>
+					your account
```

Comment:
Should this be translatable?

Reply:
Yeah, that confused me too. But I think this text is not used by the `createInterpolateElement` function. The one in the original string is preferred. I confirm this by tweaking the "your account" copy in both places and see which one gets picked up. 

I could have just give a self closing `a` tag to the `createInterpolateElement` and that would work. However, that makes the linter unhappy. 

```js
a: (
	<a
		href="https://woocommerce.com/my-account/my-subscriptions"
		target="_blank"
		rel="nofollow noopener noreferrer"
	/>
),
```